File: examples/javascript/get_bounties.js
const BASE_URL = "https://wattcoin-production-81a7.up.railway.app";

async function getBounties() {
  try {
    const response = await fetch(`${BASE_URL}/bounties`);
    if (!response.ok) {
      throw new Error(`HTTP error! status: ${response.status}`);
    }
    const data = await response.json();
    console.log(data);
  } catch (error) {
    console.error("Error fetching bounties:", error);
  }
}

getBounties();

File: examples/python/get_bounties.py
import requests

BASE_URL = "https://wattcoin-production-81a7.up.railway.app"

def get_bounties():
    """Fetch active bounties."""
    response = requests.get(f"{BASE_URL}/bounties")
    response.raise_for_status()
    return response.json()

if __name__ == "__main__":
    bounties = get_bounties()
    print(bounties)
